config/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ check_include_files(langinfo.h HAVE_LANGINFO_H)
 check_include_files(libgen.h HAVE_LIBGEN_H)
 check_include_files(locale.h HAVE_LOCALE_H)
 check_include_files(pwd.h HAVE_PWD_H)
+check_include_files(string.h HAVE_STRING_H)
```

Comment:
I added it with the check for `strings.h`, but yeah, it's redundant, so I deleted it.